### DISCUSSION TITLE
A way to mark successful skill roll

### DISCUSSION THREAD

**Discussion by Akexis** [fvtt-fria-ligan/twilight2000-foundry-vtt#126] [Ideas]
To spend experience, you have to have succeeded at a roll with a skill. Is there a way this can be tracked on the character sheet? Something like a check box?

And a question - if I add actors, such as people and vehicles to the core set folder, will that get overwritten in updates?

**Comment by Stefouch** [comment]
> To spend experience, you have to have succeeded at a roll with a skill. Is there a way this can be tracked on the character sheet? Something like a check box
No except for the description field you can use to add notes.
Thus, I am turning this discussion into a feature request.

> if I add actors, such as people and vehicles to the core set folder, will that get overwritten in updates?
No they won't be overwritten. Updates only affect imported stuff based on their ID number, not the folder.

**Reply by Akexis** [reply]
Awesome, thank you! Great work by the way, you've done a phenomenal job.
